sdk/core/Azure.Core.TestFramework/src/TestProxyException.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Runtime.Serialization;
+
+namespace Azure.Core.TestFramework
+{
+    [Serializable]
+    public class TestProxyException : Exception
```

Comment:
Are we handline these in some special way?

Reply:
No, it isn't technically needed.

Reply:
I just wasn't sure of a good exception to throw in this case..

Reply:
InvalidOperationException is ok